functions/go/put-setter-values/README.md

```diff
@@ -0,0 +1,94 @@
+# put-setter-values
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Put setter values in Kptfile or functionConfig for setters.
```

Comment:
I would rephrase this for more clarity. Also the fact the the functionConfig is inlined in Kptfile or not shouldn't be the important thing, so may want to remove that.